src/sentry/relay/config/metric_extraction.py

```diff
@@ -305,7 +305,7 @@ def _is_widget_query_low_cardinality(widget_query: DashboardWidgetQuery, project
     New queries will be checked upon creation and not allowed at that time.
     """
     params: Dict[str, Any] = {
-        "statsPeriod": "1d",
+        "statsPeriod": "30m",
```

Comment:
30m or at most 1h should be all that is necessary to determine cardinalities impact on storage.